From 919f6561b30cd2ef836ca5cbee6afd83d527f4b1 Mon Sep 17 00:00:00 2001
From: Paolo Pisati <paolo.pisati@canonical.com>
Date: Tue, 23 Jan 2024 16:33:16 +0100
Subject: [PATCH 4/4] file: export file_close_fd() instead of
 close_fd_get_file()

Following the refactor done in:

commit a88c955fcfb49727d0ed86b47410f6555a8e69e4
Author: Christian Brauner <brauner@kernel.org>
Date:   Thu Nov 30 13:49:07 2023 +0100

    file: s/close_fd_get_file()/file_close_fd()/g

update the export directive we added in:

commit 17f7fa4baad268cc4a93205747222be931699788
Author: Christian Brauner <brauner@kernel.org>
Date:   Wed Jan 16 23:13:25 2019 +0100

    UBUNTU: SAUCE: binder: turn into module

Anbox probably needs a review too.

Signed-off-by: Paolo Pisati <paolo.pisati@canonical.com>
Signed-off-by: Alexandre Frade <kernel@xanmod.org>
---
 fs/file.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/file.c b/fs/file.c
index c24f36798f55..6764a779b8ef 100644
--- a/fs/file.c
+++ b/fs/file.c
@@ -823,7 +823,7 @@ struct file *file_close_fd(unsigned int fd)
 
 	return file;
 }
-EXPORT_SYMBOL(close_fd_get_file);
+EXPORT_SYMBOL(file_close_fd);
 
 void do_close_on_exec(struct files_struct *files)
 {
-- 
2.43.0

